api-tests/src/steps/ipv-steps.ts

```diff
@@ -89,7 +89,7 @@ const startNewJourney = async (
 ): Promise<void> => {
   world.userId = world.userId ?? getRandomString(16);
   world.journeyId = getRandomString(16);
-  world.featureSet = featureSet;
+  world.featureSet = featureSet || world.featureSet;
```

Comment:
(this inconsistency is a bit annoying - how much work would it be to make everything use the `I activate the feature set` step?)

Reply:
Let's find out...